src/modules/Playlists/XSPF.hpp

```diff
@@ -0,0 +1,34 @@
+/*
+	QMPlay2 is a video and audio player.
+	Copyright (C) 2010-2016  Błażej Szczygieł
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Lesser General Public License as published
+	by the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Lesser General Public License for more details.
+
+	You should have received a copy of the GNU Lesser General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <Playlist.hpp>
+
+class XSPF : public Playlist
+{
+public:
+	XSPF(const QStringList &relativeDirs, bool useRelative);
+	Entries read();
+	bool write(const Entries &);
+
+	~XSPF();
```

Comment:
I know that I placed some destructors to the bottom, but this is ugly (also my old code :smile:), so please add it under constructor and then add one line break.